home/app.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var homeMain = $('#home-main');
+var projectMain = $('#project-main');
+var mainCSS = $('#display')
+
+projectMain.hide();
+var navMenu = $($('.navbar').find('nav'));
```

Comment:
You do not need to construct a new jQuery object here, since the `.find()` method already returns a jQuery object.